app/views/layouts/_navbar.html.haml

```diff
@@ -7,20 +7,30 @@
     = active_link_to "Home", home_path, class: "item"
     = active_link_to "Images", images_path, class: "item"
 
+    - if signed_in?
+      = active_link_to "Bookmarks", bookmarks_path, class: "item"
+
     = content_for :navbar
 
     .right.menu
-      = content_for :navbar_right
+      - if signed_in?
+        .ui.item
+          = active_link_to new_bookmark_path, class: "ui inverted green icon button", class_active: "active" do
+            %i.bookmark.icon
+            new bookmark
 
-      .ui.category.search.item{ data: { behavior: "search-images" } }
-        .ui.icon.input
-          = form_tag search_images_path, method: :get do
-            = search_field_tag "q", params[:q], class: "ui prompt", placeholder: "Search Images ...", aria: { label: "Search" }
-          %i.search.icon
-        .results
+      = content_for :navbar_right
 
       - if signed_in?
-        = link_to "Logout", logout_path, class: "item"
+        .ui.simple.dropdown.item
+          %i.user.circle.outline.icon
+          = current_user.username
+          %i.dropdown.icon
+          .menu
+            = link_to "Profile", profile_path, class: "item"
+            = link_to "Admin Panel", admin_root_path, class: "item" if current_user.role? :admin
```

Comment:
last I checked, this didn't work.

that was probably several hours ago.

Reply:
:trollface: